.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Committing this seems like a bad idea as it means that people won't be able to set their own settings. Maybe a "template" file with a different name?

Reply:
Totally agree here 👍 maybe `.vscode.template` or similar? This was really helpful for me (I didn't know about `allTargets` 👀) so other people might find that nice too